### PR TITLE
fix flaking tests on 31st of the month

### LINKED ISSUE #1531: Psammead Timestamp Container test failing
**Describe the bug**

When running unit tests locally or on CI, we get this failing test:
Screen Shot failing test

**To Reproduce**
Steps to reproduce the behaviour:

1. `npm run ci:packages`
2. `npm run test`
3. See error

**Expected behaviour**
Test should pass

**Screenshots**
Screenshot above

### PR DESCRIPTION
**Pull request by andrew-nowak**
Resolves #1531

**Overall change:** There was an edge case with relative timestamps on the 31st of a month (months are defined as 30 days long). This wouldn't be affecting any pages (relative timestamps are turned off when > 10 hours) so just fix the edge case in the tests

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

**Review by jamesdonoh — APPROVED** [review]
Thanks for the speedy fix! 👍

File: packages/containers/psammead-timestamp-container/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 2.3.2 | [PR#?](https://github.com/bbc/psammead/pull/?) 'fixes' a unit test that flaked on 31st of a month |
```
Comment (dr3) [inline packages/containers/psammead-timestamp-container/CHANGELOG.md]:
```suggestion
| 2.3.2 | [PR#1532](https://github.com/bbc/psammead/pull/1532) 'fixes' a unit test that flaked on 31st of a month |
```

**Review by dr3 — APPROVED** [review]
Im Looking forward to the next Feb 29th when simorgh just deletes itself :P

**Review by pjlee11 — APPROVED** [review]

File: packages/containers/psammead-timestamp-container/src/timestampUtilities.test.js [hunk 2]
```diff
@@ -183,7 +183,7 @@ describe('Moment configuration', () => {
 
     const allButAYear = moment()
       .subtract(12, 'months')
-      .add(1, 'second');
+      .add(5, 'days');
```
Comment (andrew-nowak) [inline packages/containers/psammead-timestamp-container/src/timestampUtilities.test.js:189]:
```suggestion
      // this was originally `.add(1, 'second') but that caused the tests to fail on the 31st of the month
      // (because the threshold for a month has been configured to 30 days) so instead
      // be safely away from this edge case by going 5 days away
      .add(5, 'days');
```

**Review by sareh — APPROVED** [review]
:+1: Thanks for the fix!

**Comment by david-boydell** [comment]
This is ready for merge.
